docs/spec/src/steps/services/containeranalysis/index.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  IntegrationInstanceConfig,
+  RelationshipClass,
+  StepSpec,
+} from '@jupiterone/integration-sdk-core';
+
+export const functionSteps: StepSpec<IntegrationInstanceConfig>[] = [
+  {
+    /**
+     * ENDPOINT: https://cloud.google.com/container-analysis/docs/reference/rest/v1/projects.notes/list
+     * PATTERN: Fetch Entities
+     * REQUIRED PERMISSIONS: projects.notes.list
+     */
+    id: 'fetch-container-analysis-notes',
+    name: 'Fetch Container Analysis Notes',
+    entities: [
+      {
+        resourceName: 'Container Analysis Note',
+        _type: 'google_cloud_project_container_analysis_note',
+        _class: ['Assessment'],
+      },
+    ],
+    relationships: [],
+    dependsOn: [],
+    implemented: false,
+  },
+  {
+    /**
+     * ENDPOINT: https://cloud.google.com/container-analysis/docs/reference/rest/v1/projects.occurrences/list
+     * PATTERN: Fetch Entities
+     * REQUIRED PERMISSIONS: projects.notes.occurrences.list
+     */
+    id: 'fetch-container-analysis-occurrences',
+    name: 'Fetch Container Analysis Occurrences',
+    entities: [
+      {
+        resourceName: 'Container Analysis Occurrence',
+        _type: 'google_cloud_container_analysis_occurrence',
+        _class: ['Finding'],
+      },
+    ],
+    relationships: [],
+    dependsOn: [],
+    implemented: false,
+  },
+  {
+    /**
+     * PROPERTY: projects.noteOccurrences
+     * PATTERN: Build Child Relationships
+     * REQUIRED PERMISSIONS: n/a
+     */
+    id: 'build-container-analysis-note-occurrences-relationships',
+    name: 'Build Container Analysis -> Note Occurrence',
```

Comment:
nit: I think this is a slight typo?

```suggestion
    name: 'Build Container Analysis Note -> Occurrence',
```

Reply:
Updated! Thank you